src/util.ts

```diff
@@ -449,3 +449,28 @@ export function exec(command: string, args?: ReadonlyArray<string>, options?: cp
     });
     return disposable;
 }
+
+/**
+ * Check if an R package is available or not
+ *
+ * @param name the R package name that need to be checked
+ * @returns a boolean Promise
+ */
+export async function isRPkgIntalled(name: string, msg?: string): Promise<boolean> {
+    const cmd = `cat(suppressPackageStartupMessages(require(${name}, quietly=TRUE)))`;
```

Comment:
Looks like the method will break if user write anything to stdout in `.Rprofile`.

Reply:
You are right. Now the code is more robust. You can test this by adding the following lines to .Rprofile

```r
print("starting message.")
.Last <- function() {
  print("session end message.")
}
```